lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -743,8 +743,9 @@ def test_insufficient_data(self):
         assert len(data['warnings']) == 0
         assert len(data['general_errors']) == 1
         assert data['general_errors'][0]['response'] ==\
-               'Data in row #1 must have exactly four columns: email, username, full name, and country'
-        # lint-amnesty, pylint: disable=line-too-long
+            'Data in row #1 must have between four and six columns: '\
+            'email, username, full name, country, cohort, and course mode. '\
+            'The last two columns are optional.'
```

Comment:
Let's make this translatable, like other errors.

Reply:
@Agrendalath It's a test, so I think it doesn't need it. I'll check it.

Reply:
Yeah, I've looked at the wrong filename.